client/blocks/legal-updates-banner/style.scss

```diff
@@ -0,0 +1,46 @@
+.legal-updates-banner {
+	position: fixed;
+	bottom: 0;
+	margin: 0;
+	width: 100%;
+	z-index: z-index( 'root', '.legal-updates-banner' );
+	padding: 1rem;
+	background-color: var( --color-neutral-80 );
+	color: var( --color-text-inverted );
+	display: flex;
+	flex-direction: column;
+
+	a {
+		color: var( --color-text-inverted );
+		text-decoration: underline;
+	}
+
+	.legal-updates-banner__content {
+		max-width: 800px;
```

Comment:
Curious if this is arbitrary, or based on something?

Reply:
I used the same style guidelines we used on the GDPR banner (see p9mQOq-Hz-p2).

Reply:
![image](https://user-images.githubusercontent.com/1233880/74015016-20a4a200-4990-11ea-96cd-1e5c6d8dd5b1.png)